dev/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7.4-fpm
```

Comment:
Can we use PHP 8.1 as this is what we're running on the web server?

Reply:
Yes we can, I uploaded the version